crates/ide/src/hover/render.rs

```diff
@@ -1210,3 +1222,85 @@ fn render_dyn_compatibility(
         }
     }
 }
+
+fn is_pwr2(val: u128) -> bool {
```

Comment:
Let's inline (remove) this.

Reply:
Sorted in https://github.com/rust-lang/rust-analyzer/pull/18973/commits/5ad3de8b29840c5975ae04b4d0d5f5d85dafc179